start-prod.sh

```diff
@@ -17,6 +17,12 @@ fi
 if [[ ! -z "${RECAPTCHA_SITE_KEY}" ]]; then
     echo "var RECAPTCHA_SITE_KEY = \"${RECAPTCHA_SITE_KEY}\";" >> ${ENV_FILE}
 fi
+if [[ ! -z "${WEB_VERSION}" ]]; then
+    echo "var WEB_VERSION = \"${WEB_VERSION}\";" >> ${ENV_FILE}
+fi
+if [[ ! -z "${WEB_BUILD}" ]]; then
+    echo "var WEB_BUILD = \"${WEB_BUILD}\";" >> ${ENV_FILE}
+fi
```

Comment:
@rkorytkowski I tried this:
1. Added a static version on package.json in the "app_version" key's value
2. Updated "version" key in package.json to hold SHA or "dev", so that `npm version ${COMMIT_SHA:-dev}` can work -- But this is not allowed. "version" key has to follow semantic schema (major.minor.maintenance) so it has to hold the app's version.

What do you think about:
1. using "version" key to hold app version (2.0.1) in package.json
2. adding "build" key to hold SHA or "dev" in package.json
3. Will have to use `json` or `jq` tool to update "build" in package.json, don't think npm has a command to setup a variable's value (other than version), _OR_
4. use version to hold app version and SHA together like '2.0.1-dev' and use just update post `-` part in deployment script?

Reply:
I think separating version and SHA in package.json is the right approach. Is the "build" key in package.json schema?

Reply:
I don't see the "build" key in official docs. Maybe better to have package.json just store the static version and a full version with SHA be in some property file in the app, which is easy to read and set from bash. Ideally we have a set_project_version.sh script, which updates version in both places and can be easily called from Dockerfile before running the js build.

Reply:
@rkorytkowski I have added config.json with build key that can be set via bash using `jq` or something similar. package.json now just has app version (in `version` key)